src/todos.ts

```diff
@@ -0,0 +1,20 @@
+import { Todo } from './types/Todo';
+import { client } from './utils/fetchClient';
+
+type CreatedTodoArgs = Omit<Todo, 'id'>;
+
+export const getTodos = (userId: number): Promise<Todo[]> => {
+  return client.get<Todo[]>(`/todos?userId=${userId}`);
+};
+
+export const createTodo = (data: CreatedTodoArgs) => {
+  return client.post<Todo>('/todos', data);
+};
+
+export const removeTodo = (todoId: number) => {
+  return client.delete(`/todos/${todoId}`);
+};
+
+export const updateTodo = ({ id, ...todoData } : Todo) => {
```

Comment:
you shouldn't work with a server in this task
save your todos in localStorage

![image](https://github.com/mate-academy/react_todo-app/assets/86923089/20e0e8d3-0dc0-450d-8d9c-4a7071c428e6)